packages/starknet-snap/test/src/executeTxn.test.ts

```diff
@@ -180,6 +238,7 @@ describe('Test function: executeTxn', function () {
     const stub = sandbox.stub(utils, 'executeTxn').resolves({
       transaction_hash: 'transaction_hash',
     });
+
```

Comment:
```suggestion
sandbox.stub(utils, 'isAccountDeployed').resolves(true);
```
add back account deployed is true to bypass mock

Reply:
@stanleyyconsensys thanks! I noticed I had forgotten on branch case (executeTxn multiple with account not deployed). I followed your suggestion in my last commit and also added two branches : 

`when account is not deployed` and `when account is deployed`


tests looks like this now : 

```
  Test function: executeTxn
    ✔ should throw error if executeTxn fail (63ms)
    ✔ should return false if user rejected to sign the transaction (59ms)
    when account is not deployed
      ✔ should executeTxn correctly and deploy an account (91ms)
      ✔ should executeTxn multiple and deploy an account (89ms)
    when account is not deployed
      ✔ should executeTxn and not deploy an account (63ms)
      ✔ should executeTxn multiple and not deploy an account (63ms)
```